operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -838,23 +835,42 @@ spec:
     **Streams for Apache Kafka** is a massively scalable, distributed, and high performance data streaming platform based on the Apache Kafka® project.
     Streams provides an event streaming backbone that allows microservices and other application components to exchange data with extremely high throughput and low latency.
 
-    **From the 2.6.0 release, Streams supports only OCP 4.12 to 4.14!
+    **From the 2.8.0 release, Streams supports only OCP 4.12 to 4.14!
 
     **The core capabilities include:**
     * A pub/sub messaging model, similar to a traditional enterprise messaging system, in which application components publish and consume events to/from an ordered stream
     * The long term, fault-tolerant storage of events
     * The ability for a consumer to replay streams of events
     * The ability to partition topics for horizontal scalability
-
+  
     ### Upgrades
 
     **!!! IMPORTANT !!!**
 
-    **Direct upgrade from Streams 1.7 or earlier is not supported!**
+    **Direct upgrade from Streams 1.7 or earlier is not supported anymore!**
     You have to upgrade first to one of the previous versions of Streams.
     You will also need to convert the CRD resources.
     For more information, see the [documentation for Streams for Apache Kafka on OpenShift](https://docs.redhat.com/en/documentation/red_hat_streams_for_apache_kafka/).
 
+    **Strimzi 0.43 is the last version with support for Kubernetes 1.23 and 1.24.**
+    From Strimzi 0.44, only Kubernetes versions 1.25 and newer will be supported.
+
+    ### New in 2.8
+
+    * Support for Apache Kafka 3.8.0
+
+    * Removed support for Apache Kafka 3.6.0, 3.6.1, and 3.6.2
+
+    * Specifying additional volumes and volume mounts in Streams custom resources
+
+    * Support for custom Cruise Control API users
+
+    * Unregistration of KRaft nodes after scale-down
+
+    * Kafka Exporter updated to 1.8.0 (and the Grafana dashboard updated to work with it)
+
+    * Service account based authentication
+
```

Comment:
We can remove this block, it is specific to 0.43.0 from what I understand, not the range of 0.39.0-0.43.0

Reply:
Removed